ovos_persona/__init__.py

```
@@ -44,32 +47,40 @@ def __init__(self, name, config, blacklist=None):
-            if plug_name not in solver_order or plug_name in blacklist:
+
+        memory_plugin = self.config.get("memory_module", "ovos-agents-short-term-memory-plugin")
+        if memory_plugin:
+            memory_class = load_memory_plugin(memory_plugin)
+            self.memory = memory_class()
+        else:
+            self.memory = None
+
+        plugin_order = config.get("handlers") or config.get("solvers") or []
+        if not plugin_order:
+            raise ValueError("personas must have at least 1 utterance handler")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)